src/components/Timeline/src/Timeline.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import MaterialStepper from '@material-ui/core/Stepper';
+import BaseProps from '@gemeente-denhaag/baseprops';
+
+import { timelineClasses } from './bem-mapping';
+
+export interface TimelineProps extends BaseProps {
```

Comment:
I think we should prepend interface names with `I` for clarity. So this interface would become `ITimelineProps`. That way it is much easier to distinguish what's what and prevents devs from expecting a component where there is none.

Reply:
We have never prefixed our interfaces with `I`.

Reply:
Renaming them is not that difficult, most IDEs have easy rename functions and often do automatic refactoring as well. I know in WebStorm one can press Shift+F6 to rename a c;lass or interface and it automatically refactors almost everything from imports to inherits to package.jsons

Reply:
I know, but no other interface we have is prefixed with `I`